capsules/src/console.rs

```diff
@@ -255,25 +254,13 @@ impl<'a, U: UART> Driver for Console<'a, U> {
                 self.apps.enter(app_id, |app, _| {
                     app.write_callback = callback;
                     ReturnCode::SUCCESS
-                }).unwrap_or_else(|err| {
-                    match err {
-                        Error::OutOfMemory => ReturnCode::ENOMEM,
-                        Error::AddressOutOfBounds => ReturnCode::EINVAL,
-                        Error::NoSuchApp => ReturnCode::EINVAL,
-                    }
-                })
+                }).unwrap_or_else(|err| err.into())
```

Comment:
These are just generally good and could probably move into their own, independent PR

Reply:
Please do!

Reply:
Done, #959 :)